styles.css

```diff
@@ -18,4 +22,579 @@ section {
+  font-size: 14px;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.program-items {
+  display: flex;
+  margin: 45px 0;
+  flex-direction: column;
+  gap: 5px;
+}
+
+.program-item {
+  padding: 10px;
+  background-color: rgba(255, 255, 255, 0.1);
+  text-align: left;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.program-item h3 {
+  font-size: 20px;
+  color: var(--color2);
+  font-weight: 700;
+}
+
+.program-item img {
+  margin: 5px;
+  width: 15%;
+}
+
+.program-item p {
+  width: 50%;
+}
+
+.speakers,
+.about-2 {
+  width: 100%;
+  max-width: 870px;
+  margin: 0 auto;
+  padding: 40px 25px;
+}
+
+.speaker-items {
+  margin: 40px 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.speaker-item {
+  margin: 15px 0;
+  display: flex;
+  align-items: flex-start;
+  line-height: 1.5;
+}
+
+.speaker-item h3 {
+  font-size: 20px;
+  color: var(--color1);
+  font-weight: 400;
+  margin: 0;
+  padding: 0;
+}
+
+.speaker-item h4 {
+  color: var(--color2);
+  font-style: italic;
+  margin: 0;
+  padding: 0 0 10px 0;
+  background: url(images/speakers-h4.png) bottom left no-repeat;
+}
+
+.speaker-image {
+  width: 35%;
+}
+
+.speaker-image img {
+  max-width: 100%;
+}
+
+.speaker-details {
+  padding: 0 0 0 20px;
+  width: 65%;
+}
+
+.partners-container {
+  width: 100%;
+  padding: 40px 0;
+  background-color: #414244;
+  color: var(--color3);
+}
+
+.partners {
+  width: 100%;
+  padding: 0 25px;
+}
+
+.partner-logos {
+  margin: 30px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.partner-logos img {
+  width: 25%;
+  margin: 10px;
+}
+
+footer,
+.footer-content {
+  margin: 0 auto;
+  padding: 50px 25px;
+  width: 100%;
+  max-width: 870px;
+  font-size: 12px;
+  text-align: center;
+  line-height: 1.5;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-logo {
+  margin: 10px 0;
+}
+
+.about-body {
+  background-color: #fff;
+}
+
+.about-container {
+  border-bottom: 1px solid var(--color3);
+}
+
+.about {
+  padding-bottom: 50px;
+  text-align: center;
+}
+
+.about h1 {
+  padding-bottom: 20px;
+  margin-bottom: 30px;
+  max-width: 100%;
+  background: url(images/h2-border.png) center bottom no-repeat;
+}
+
+.about-content {
+  max-width: 100%;
+  background-color: #fff;
+  border: 1px solid #d3d3d3;
+}
+
+.about-2 {
+  text-align: center;
+  background-color: #fff;
+}
+
+.about2-content {
+  border: 0;
+}
+
+.about2-content-img {
+  margin: 30px auto;
+  padding: 20px;
+  width: 100%;
+  max-width: 450px;
+  border: 1px solid var(--color3);
+}
+
+.about-3-container {
+  border-top: 1px solid var(--color3);
+}
+
+.about-3-gallery {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.gallery {
+  width: 100%;
+  padding: 10px;
+}
+
+.gallery img {
+  max-width: 100%;
+}
+
+.inner-footer {
+  padding: 0 25px;
+  max-width: 100%;
+  background-color: var(--color1);
+}
```

Comment:
- [ ] Kindly make sure the footer in the "About page" of the mobile version has the proper background color as shown in the original design below 👇 this will improve the design consistency in relation to the Behance design. The color should be `#FFFFFF` and not `#272a31` this is only for the mobile version.
- Your design:
  ![image](https://github.com/balatstar/capstone1/assets/104968236/57c80c4e-c5ef-4d2e-8f78-ba3a7fd677b1)
- Behance design:
  ![image](https://github.com/balatstar/capstone1/assets/104968236/997ea224-e710-4f65-b4a3-a7f558584ae1)